### PR TITLE
Dwh 2021 create directories

### PR DESCRIPTION
**Pull request by sofiabravo103**
Added script to re create sql files inside directories that will represent a pipeline each while also adding the configuration file.

### CODE REVIEW

File: bimaterializedviewsautomation/sql_cehcks/sanity_checks.py [hunk 1]
```diff
@@ -7,7 +7,12 @@
```
Comment (nazsal) [inline bimaterializedviewsautomation/sql_cehcks/sanity_checks.py]: I think we can merge this one, but there won't be `is_incremental` in `bimaterializedviewsautomation.parsing_utils`  anymore so we have to fix it as well

**Comment by nazsal** [comment]
+1

File: bimaterializedviewsautomation/sql_cehcks/validate_in_hive.py [hunk 2]
```diff
@@ -36,31 +37,23 @@ def execute(self, explain_query):
...
         return True
 
 
-path = os.path.join(os.path.dirname(__file__), '../../materialized_views/')
-
-for filename in os.listdir(path):
-    with open(os.path.join(path, filename)) as matview:
-        script_text = matview.read()
-
+def hive_validation(script_text):
```
Comment (siklosid) [inline bimaterializedviewsautomation/sql_cehcks/validate_in_hive.py]: why don't we have a function called validation which based on the configuration decides which validaton to call?

Comment (sofiabravo103) [reply bimaterializedviewsautomation/sql_cehcks/validate_in_hive.py]: Seems more complicated to do it that way, what would be the benefit?

Comment (siklosid) [reply bimaterializedviewsautomation/sql_cehcks/validate_in_hive.py]: for me it seems simpler.
It's easier to read what's happening if I don't have to know that somewhere from the outside both validation function is called and if the configuration doesn't match then the other function will run.

Comment (sofiabravo103) [reply bimaterializedviewsautomation/sql_cehcks/validate_in_hive.py]: Ok, I'll change the code.

File: bimaterializedviewsautomation/manual_scripts/recreate_pipelines_directories.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    mat_views = {}
+    mat_views_path = os.path.join(VIEWS_BASE, VIEWS_DB)
+    for script_file_name in os.listdir(mat_views_path):
+        script_file_full_path = os.path.join(mat_views_path, script_file_name)
+        if os.path.isdir(script_file_full_path):
+            continue
+        with open(script_file_full_path, "r") as query_file:
+            text = query_file.read()
+        incremental = is_incremental(text)
+        is_impala = is_marked_for_impala(text)
```
Comment (nazsal) [inline bimaterializedviewsautomation/manual_scripts/recreate_pipelines_directories.py]: I also removed this method from parsing utils

**Comment by siklosid** [comment]
+1

**Comment by siklosid** [comment]
+1

**Pull request merged**